prow/config.yaml

```diff
@@ -515,9 +515,9 @@ presets:
       configMapKeyRef:
         name: release-1.4-deps
         key: dependencies
-# Enable GOPROXY by default
+# Enable GOPROXY by default and fallback to direct connection.
 - env:
   - name: GOPROXY
-    value: "https://proxy.golang.org"
+    value: "https://proxy.golang.org,direct"
```

Comment:
This is already the default. I don't think its a fallback here. We do get `@master`, which exists, its just stale. The fallback would be if we did `@non-existent`

Reply:
It is my understanding that in Go@1.13 that `"https://proxy.golang.org,direct"` is the **default** when the `GOPROXY` variable is *unspecified* ([ref](https://golang.org/doc/go1.13#modules)). Before it was being explicitly set to `"https://proxy.golang.org"`. If this is not the case (or the solution), I can additionally install using the SHA.

Reply:
Ah yes, good point. I forgot about this. sgtm

Reply:
I just pushed additional changes to do `go get istio.io/api@SHA` rather than branch.  My hope is that this, combined with falling back to *direct* will fix the race 🤞.